src/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { Checkbox as RKCheckbox } from "reakit/Checkbox";
+import { colors } from "../shared/constants";
+import styled, { css } from "styled-components";
+
+export interface CheckboxProps {
+  disabled?: boolean;
+  checked?: boolean;
+  toggle?: Function;
+  [key: string]: any;
```

Comment:
Let's add the prop `label` here as well. Which will be the text.

Requirement:
- Clicking on text should toggle the checkbox

Reply:
Added! As well as passed children incase we need more flexibility